.eslintrc.json

```diff
@@ -0,0 +1,3 @@
+{
+  "extends": "next/core-web-vitals"
```

Comment:
👍 linting rules
We have our own internal configurations that we use for most projects